CHANGELOG.md

```diff
@@ -243,88 +243,3 @@ All notable changes to this project will be documented in this file. See [standa
 * deprecated reset() method is now private (call yargs() instead). ([376f892](https://www.github.com/yargs/yargs/commit/376f89242733dcd4ecb8040685c40ae1d622931d))
 * implicitly private methods are now actually private ([376f892](https://www.github.com/yargs/yargs/commit/376f89242733dcd4ecb8040685c40ae1d622931d))
 * **yargs-factory:** refactor yargs-factory to use class ([#1895](https://www.github.com/yargs/yargs/issues/1895)) ([376f892](https://www.github.com/yargs/yargs/commit/376f89242733dcd4ecb8040685c40ae1d622931d))
```

Comment:
For interest, was there any particular reason for archiving the older changelogs? 

(I did this myself in Commander because CHANGELOG was included in the published package, but npm no longer does that so I shifted it back to a single file. I like being able to search for what versions changes shipped in, but it is only a minor convenience.)

Reply:
I was doing it to keep the bundle size down, I didn't realize npm no longer published CHANGELOG.md.

Reply:
went ahead and recombined the CHANGELOGs given this feedback.